src/sip.c

```diff
@@ -626,18 +647,20 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
         }
 
         // Check if we have attribute format RTCP port
-        if (!strncmp(line, "a=rtcp:", 7)) {
-            if (media && sscanf(line, "a=rtcp:%hu", &dst.port)) {
-                // Create early RTCP stream
-                if (!rtp_find_call_stream(call, src, dst)) {
-                    call_add_stream(call, stream_create(media, dst, PACKET_RTCP));
-                }
-            }
+        if (!strncmp(line, "a=rtcp:", 7) && rtcp_stream) {
+            sscanf(line, "a=rtcp:%hu", &rtcp_stream->dst.port);
```

Comment:
We don't have to create a new stream, just assign the given port to the current RTCP one.